src/ocean/io/stream/Buffered.d

```diff
@@ -1166,6 +1166,18 @@ public class BufferedOutput : OutputFilter, OutputBuffer
         return this;
     }
 
+    /***************************************************************************
+
+        Flushes the buffer and closes the stream.
+
+    ***************************************************************************/
+
+    final override void close ( )
```

Comment:
Would be nice to add a unittest for this.

Reply:
That would be inherently flakey because you don't control buffering afaics.

Reply:
Hmm, is there some way that the buffer would be flushed magically behind the scenes?

I was thinking of something like:
1. Open file with buffered output.
2. Write something smaller than the buffer size.
3. Close the file.
4. Check that the file content on disk is correct. (i.e. the buffer was flushed.)

Reply:
Yeah, makes sense. Added.